schemas/deviceShadow/ipShadow/ipShadow-example.json

```diff
@@ -52,8 +52,6 @@
             "TEMP",
             "HUMID",
             "AIR_PRESS",
-            "BUTTON",
-            "LIGHT",
```

Comment:
Just curious why you removed "BUTTON" and "LIGHT" here.

Reply:
We don't have cards on the FE for these. So if the device sends messages using BUTTON or LIGHT appIds they will only be displayed on the terminal. 
There is a Light card planned for the future https://projecttools.nordicsemi.no/jira/browse/IRIS-2159

Reply:
Hmm, that ticket is 2 years old...

Reply:
Isn't it still valid to send such data? I see ATv2 sends "BUTTON" when I press the center button on the Thingy91, and I can see it in the Terminal card, which is useful. So IMHO we should allow any env sensor a device could send, including these.

Reply:
What you are seeing in the terminal card are messages which is fine. The device can send anything they want and it'll be shown on the terminal card. This is a doc for the device shadow. This part is telling the user that the frontend will only show cards for [these](https://github.com/nRFCloud/application-protocols/pull/28/files/81a8a3cd07638c3dae8b7536a490ffdb2ccf26b4#diff-579a91a83ad8aedf70952a99c2d3856ca33ac3c7542be9a242184865e2c6a54dR291) appIds. There are no cards for button and light, so why would I show that in this example?